js/localization/messages/tr.json

```
@@ -0,0 +1,261 @@
+        "dxScheduler-editorLabelRecurrence": "Tekrar",
+        "dxScheduler-openAppointment": "Randevu Aç",
+        "dxScheduler-recurrenceNever": "Asla",
+        "dxScheduler-recurrenceDaily": "Günlük",
+        "dxScheduler-recurrenceWeekly": "Haftalık",
+        "dxScheduler-recurrenceMonthly": "Aylık",
+        "dxScheduler-recurrenceYearly": "Yıllık",
+        "dxScheduler-recurrenceRepeatEvery": "Her tekrarla",
+        "dxScheduler-recurrenceRepeatOn": "Tekrarla",
+        "dxScheduler-recurrenceEnd": "Tekrarı bitir",
+        "dxScheduler-recurrenceAfter": "Sonra",
+        "dxScheduler-recurrenceOn": "İle",
+        "dxScheduler-recurrenceRepeatDaily": "günler",
+        "dxScheduler-recurrenceRepeatWeekly": "haftalar",
+        "dxScheduler-recurrenceRepeatMonthly": "aylar",
+        "dxScheduler-recurrenceRepeatYearly": "yıllar",
+        "dxScheduler-switcherDay": "Gün",
+        "dxScheduler-switcherWeek": "Hafta",
+        "dxScheduler-switcherWorkWeek": "Çalışma Haftası",
+        "dxScheduler-switcherMonth": "Ay",
+        "dxScheduler-switcherAgenda": "Ajanda",
+        "dxScheduler-switcherTimelineDay": "Zaman Çizelgesi Günü",
+        "dxScheduler-switcherTimelineWeek": "Zaman Çizelgesi Haftası",
+        "dxScheduler-switcherTimelineWorkWeek": "Zaman Çizelgesi Çalışma Haftası",
+        "dxScheduler-switcherTimelineMonth": "TZaman Çizelgesi Çalışma Ayı",
+        "dxScheduler-recurrenceRepeatOnDate": "tarihinde",
+        "dxScheduler-recurrenceRepeatCount": "olaylar",
+        "dxScheduler-allDay": "Tüm gün",
+        "dxScheduler-confirmRecurrenceEditMessage": "Yalnızca bu randevuyu veya tüm diziyi düzenlemek ister misiniz?",
+        "dxScheduler-confirmRecurrenceDeleteMessage": "Yalnızca bu randevuyu veya tüm diziyi silmek istiyor musunuz?",
+        "dxScheduler-confirmRecurrenceEditSeries": "Serileri düzenle",
+        "dxScheduler-confirmRecurrenceDeleteSeries": "Serileri sil",
+        "dxScheduler-confirmRecurrenceEditOccurrence": "Randevuyu düzenle",
+        "dxScheduler-confirmRecurrenceDeleteOccurrence": "Randevuyu sil",
+        "dxScheduler-noTimezoneTitle": "Saat dilimi yok",
+        "dxScheduler-moreAppointments": "{0} data",
+        "dxCalendar-todayButtonText": "Bugün",
+        "dxCalendar-ariaWidgetName": "Takvim",
+        "dxColorView-ariaRed": "Kırmızı",
+        "dxColorView-ariaGreen": "Yeşil",
+        "dxColorView-ariaBlue": "Mavi",
+        "dxColorView-ariaAlpha": "Şeffaflık",
+        "dxColorView-ariaHex": "Renk kodu",
+        "dxTagBox-selected": "{0} seçili",
+        "dxTagBox-allSelected": "Tümü seçildi ({0})",
+        "dxTagBox-moreSelected": "{0} daha",
+        "vizExport-printingButtonText": "Yazdır",
+        "vizExport-titleMenuText": "Dışa Aktar/Yazdır",
+        "vizExport-exportButtonText": "{0} dosya",
+        "dxFilterBuilder-and": "Ve",
+        "dxFilterBuilder-or": "Veya",
+        "dxFilterBuilder-notAnd": "Değil",
```

Is it better to use this phrase here?
```suggestion
        "dxFilterBuilder-notAnd": "Değil Ve",
```